### DISCUSSION TITLE
Use ReadConcern along with MongoDB Panache

### DISCUSSION THREAD

**Discussion by md-arman-ahold** [quarkusio/quarkus#37076] [Q&A]
Hello,
I want to use Read Concern "Snapshot" when streaming data from MongoDB. The current implementation uses MongoDB panache for the operation. How may I use the ReadConcern along with the MongoDB panache? Any suggestions/example?

[a comment by an automated account is not shown]

**Comment by loicmathieu** [comment]
Hi,
You can set read concern at the MongoDB client level by using the following configuration property: https://quarkus.io/guides/mongodb#quarkus-mongodb_quarkus.mongodb.read-concern
